Goobi/src/org/goobi/production/properties/ProcessProperty.java

```diff
@@ -59,9 +59,12 @@
 	private boolean currentStepDuplicationAllowed = false;
 	
 	public ProcessProperty() {
+		this.container = new Integer(0);
```

Comment:
Sorry, that was a misunderstanding on my side. It's fixed now with an updated PR.